src/Credentials.js

```diff
@@ -1,5 +1,10 @@
-import { createJWT, verifyJWT } from './JWT'
-import { decodeToken } from 'jsontokens'
+import { SimpleSigner, decodeJWT, createJWT, verifyJWT } from 'did-jwt'
+const MNID = require('mnid')
+import { ec as EC } from 'elliptic'
+const secp256k1 = new EC('secp256k1')
+
+import UportDIDResolver from 'uport-did-resolver'
```

Comment:
Make sure to add the `ethr-did-resolver` as well. As it should be able to consume ethr-dids as well